.github/workflows/terraform.yml

```diff
@@ -41,6 +41,11 @@ jobs:
           echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
           chmod +x "$GITHUB_WORKSPACE/bin/argocd"
 
+      - name: Setup terraform code dependencies
+        run: |
+          wget https://amazon-eks.s3.us-west-2.amazonaws.com/1.19.6/2021-01-05/bin/linux/amd64/aws-iam-authenticator -O "$GITHUB_WORKSPACE/bin/aws-iam-authenticator"
+          chmod +x "$GITHUB_WORKSPACE/bin/aws-iam-authenticator"
+
```

Comment:
this is actually not required as this workflow is only used in integration tests of the DevOps stack that supports only k3s on Docker right now.

Reply:
Yes, I hoped this would fix the Github tests, but it doesn't. I'll delete the commit form the PR.